src/renderer/src/lib/Music.ts

```diff
@@ -139,6 +148,11 @@ export function pause() {
   player.pause();
 }
 
+export async function changeAudioDevice(deviceId: string) {
+  await window.api.request("settings::write", "audioDeviceId", deviceId);
+  (player as any).setSinkId(deviceId); // unsafe, but works
```

Comment:
Can you change it to proper check? (This code snippet is not tested)
```javascript
if ("setSinkId" in player && typeof player.setSinkId === "function") {
    // ...
}
```

Reply:
you can do THAT without everything falling apart? really cool!